Fakes/OpenActive.FakeDatabase.NET/FakeBookingSystem.cs

```diff
@@ -644,118 +644,6 @@ public async Task<(bool, ClassTable, OccurrenceTable, BookedOrderItemInfo)> GetO
             }
         }
 
-        public OpenActive.NET.Place GetPlaceById(long placeId)
```

Comment:
Moved this to FeedGenerationHelper as it doesn't really belong in FakeBookingSystem